sailor/pai/alert.py

```diff
@@ -137,6 +140,39 @@ def as_df(self, columns: Iterable[str] = None, include_all_custom_properties=Fal
 
         return super().as_df(columns=list(columns))
 
+    def plot_overview(self):
+        """
+        Plot an overview over all alerts in the set as a function of time.
+
+        Each alert will be shown by a point, on a y-scale representing the affected equipment
+        and with a color representing the alert type. The size of the point is given by the ``count`` value.
+        This value represents how many times the alert has occurred in the deduplication window specified by the
+        alert type.
+
+        Example
+        -------
+        Plot an overview over all alerts in the dataset "alert_set" by time::
+
+            alert_set.plot_overview()
+        """
+        data = self.as_df(columns=['last_occured_on', 'equipment_name', 'type', 'count'])
+
+        # if there are any `NA` values in the equipment_name the plot gets messed up.
+        # this turns the NAs into an 'nan' string, which works fine.
+        data['equipment_name'] = data['equipment_name'].astype(str)
```

Comment:
can any other field (`type` maybe?) be empty/NA?

Reply:
no. type is a mandatory field